### PR TITLE
fix(1650): change the way old document audit events are shown

### PR DESCRIPTION
**Pull request by afcalderonm**
# Basics

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Styling and layout changes
- [ ] Code style changes (formatting, local variables)
- [ ] Refactoring (no functional changes)
- [ ] Build or CI related changes
- [ ] Tests
- [ ] Other... Please describe:

## Screenshot of the affected changes

before change: 
image

after change:
image

## Checklist

- [ ] I have updated the documentation accordingly
- [ ] I have confirmed all acceptance criteria defined by the story have been met
- [ ] I have confirmed that Sonar Qube findings for new code are reviewed and fixed, or will be whitelisted

### CODE REVIEW

File: libs/shared/data-access/audit-api/src/lib/models/audit-event/audit-event.model.ts [hunk 1]
```diff
@@ -97,7 +97,8 @@ export interface EventUpdates {
   oldState: string;
   newState: string;
   label: string;
-  documentId: string | undefined;
+  newDocumentId: string | undefined;
+  oldDocumentId: string | undefined;
```
Comment (davidyoung77) [inline libs/shared/data-access/audit-api/src/lib/models/audit-event/audit-event.model.ts:101]: This is the more standard way of doing this in typescript
```suggestion
  newDocumentId?: string;
  oldDocumentId?: string;
```

File: libs/agency/feature/dashboard/src/lib/components/events-log/event-types/transaction-data-changed/transaction-data-changed.component.spec.ts
Comment (davidyoung77) [inline libs/agency/feature/dashboard/src/lib/components/events-log/event-types/transaction-data-changed/transaction-data-changed.component.spec.ts:1]: We should add a test here to check that the dom rendered correctly

**Review by davidyoung77 — COMMENTED** [review]
The code looks good, I have a couple minor comments

**Comment by davidyoung77** [comment]
All the screenshots are displaying file, I believe this should be displaying the filename

**Comment by asultan001** [comment]
@davidyoung77 I don't think we need to show the filename at this time because I worry that people may upload files with poor names. I think it is ok to keep it like this for now and we can enhance it in the future as part of an engagement if they want.

**Comment by afcalderonm** [comment]
> All the screenshots are displaying file, I believe this should be displaying the filename

it was intended to be like that https://dol-ewf.atlassian.net/browse/DSG-1650

File: libs/agency/feature/dashboard/src/lib/components/events-log/event-types/transaction-data-changed/transaction-data-changed.component.spec.ts
Comment (afcalderonm) [reply libs/agency/feature/dashboard/src/lib/components/events-log/event-types/transaction-data-changed/transaction-data-changed.component.spec.ts:1]: @davidyoung77  i tried to write the test but im not really that into the front-end testing, can you help me with that? or can we leave like this for now so we can close the story

**Review by davidyoung77 — APPROVED** [review]
LGTM approved

**Pull request merged**